### PR TITLE
added alert comp

### PR DESCRIPTION
**Pull request by prateek1361**

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+}
+.main-container{
+    display: flex;
+}
+
+.alert{
+    height:5rem;
+    width:50rem;
+}
+.primary{
```
Comment (surbhi84) [inline main.css:30]: you can also add more properties such as padding, margins and you can center your text as well, checkout chakra materialUi or bootstrap for reference.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    --primary:skyblue;
+    --error:red;
+    --warning:yellow;
+    --success:green;
+    --secondary:white;
+    font-size: 10px;
+
+}
+body{
+    font-size: 10px;
```
Comment (surbhi84) [inline main.css:11]: suggestion: you can also make variables for font sizes in order to maintain consistency.

**Review by surbhi84 — CHANGES_REQUESTED** [review]
the component can get better on the design front, rest of the code looks good to me.

**Review by ishitakeshawani — APPROVED** [review]
looks good to me

**Pull request merged**